### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by Adelrisk** [anymaniax/orval#879]
## What are the steps to reproduce this issue?

1. Take the following content [openapi.txt](https://github.com/anymaniax/orval/files/11755560/openapi.txt) saved as `openapi.json` (with 2862 lines)
2. Run  orval with config:

```
import { defineConfig } from "orval";

export default defineConfig({
  petStore: {
    output: {
      target: "api/endpoints/petStore.ts",
      schemas: "api/model",
      client: "swr",
    },
    input: {
      target: "./openapi.json",
    },
  },
});

```

## What happens?

It fails with the following output:

```log
🍻 Start orval v6.16.0 - A swagger client generator for typescript
🛑  petStore - RangeError: Maximum call stack size exceeded
```
## What were you expecting to happen?

The various generated files and modes.

## Any logs, error output, etc?

See above.

## Any other comments?

May be a regression of https://github.com/anymaniax/orval/issues/110

Does not work with a yaml schema either.

## What versions are you using?

**Operating System:** Linux ffs 6.1.0-9-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.27-1 (2023-05-08) x86_64 GNU/Linux
**Package Version:** npm 8.19.2 and yarn 3.5.1
**Browser Version:** N/A

**Comment by Adelrisk** [comment]
tl;dr: Fixed after updating all other dependencies and changed typescript target from 'es5' to 'es6'.

In long ... I'd given up on this tool for a while, but kept trying to use it. In the last few weeks, we've upgraded the dependencies and migrated the app from Nextjs (pages router) to Nextjs (app router).  `orval` did not break, but reported multiple times:

```log
✘ [ERROR] Transforming const to the configured target environment ("es5") is not supported yet                        
                                                                                                                      
    api/mutator/custom-instance.ts:3:7:                    
      3 │ export const DEFAULT_VALUES: RequestInit = { 
```

After changing the target in `tsconfig.json` from `es5` to `es6`, orval now reports success `Your OpenAPI spec has been converted into ready to use orval!`
